pages/establishment/pdf/views/establishment.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import map from 'lodash/map';
+import StandardConditions from '../../../common/views/pdf/standard-conditions';
+
+const getName = person => `${person.firstName} ${person.lastName}`;
+
+const triggerContinued = () => {
+  let first = true;
+
+  return () => {
+    if (first) {
+      first = false;
+      return '';
+    }
+    return ' (continued)';
+  };
+};
+
+const Establishment = ({ establishment, content }) => {
+  const licenceHolders = establishment.namedPeople.pelh || establishment.namedPeople.nprc || [];
+  const hasAdditionalConditions = !!establishment.conditions;
+  const hasApprovedAreas = !!establishment.places;
+  const hasAuthorisations = establishment.authorisations && establishment.authorisations.length > 0;
+  const killing = establishment.authorisations.filter(({ type }) => type === 'killing');
+  const rehomes = establishment.authorisations.filter(({ type }) => type === 'rehomes');
+
+  const continued = triggerContinued();
+
+  return (
+    <Fragment>
+      <div className="logo"></div>
+
+      <h3 className="licence">ESTABLISHMENT LICENCE</h3>
+
+      <h1 className="establishment-name">{establishment.name}</h1>
+      <ReactMarkdown>{establishment.address}</ReactMarkdown>
+
+      <h4>Licenced to carry out:</h4>
+      <ul>
+        {
+          ['procedure', 'breeding', 'supplying'].filter(auth => establishment[auth]).map(auth =>
+            <li key={auth}>{content.licenced[auth]}</li>
+          )
+        }
+      </ul>
+
+      <section className="licence-holder">
+        <h2>Establishment licence holder(s)</h2>
+        {
+          licenceHolders && licenceHolders.length > 0 ? (
+            <p><strong>{licenceHolders.map(getName).join(', ')}</strong></p>
+          ) : (
+            <p>None.</p>
+          )
+        }
+      </section>
+
+      <section className="named-people break">
+        <h2>Named people</h2>
+
+        <h4>Named Person Responsible for Compliance</h4>
+        <p>{ establishment.namedPeople.nprc.map(getName).join(', ') || 'None.' }</p>
+
+        <h4>Named Information Officers</h4>
+        <p>{ establishment.namedPeople.nio.map(getName).join(', ') || 'None.' }</p>
+
+        <h4>Named Training and Competency Officers</h4>
+        <p>{ establishment.namedPeople.ntco.map(getName).join(', ') || 'None.' }</p>
+
+        <h4>Named Veternary Surgeons</h4>
+        <p>{ establishment.namedPeople.nvs.map(getName).join(', ') || 'None.' }</p>
+
+        <h4>Named Animal Care and Welfare Officers</h4>
+        <p>{ establishment.namedPeople.nacwo.map(getName).join(', ') || 'None.' }</p>
+      </section>
+
+      {/* page break */}
```

Comment:
Is this _actually_ creating a page break, or purely indicative?

Reply:
Just a comment

Reply:
Class ‘.break’ adds the page break after the element it’s applied to